changelog/9780.doc.md

```diff
@@ -0,0 +1 @@
+Updates [quick install documentation](https://rasa.com/docs/rasa/installation#quick-installation) with optional venv step, better pip install instructions, & M1 warning
```

Comment:
This uses relative links (e.g. better if we consider publishing the docs under a different URL at some point) and _I think_ also considers the used version
```suggestion
Updates [quick install documentation](installation.mdx#quick-installation) with optional venv step, better pip install instructions, & M1 warning
```